File: trunk/C#/Code/SPTemplate.EventHandlers/SampleEventHandler.cs
using System;
using System.Collections.Generic;
using System.Text;
using Microsoft.SharePoint;
using System.Web;
using $customUsing$.Common;
using $customUsing$.Exceptions;

namespace  $safeprojectname$
{
    public class SampleEventHandler : SPItemEventReceiver
    {
        HttpContext current;
        public SampleEventHandler()
        {
           current = HttpContext.Current;
        }

        public override void ItemAdding(SPItemEventProperties properties)
        {
            base.ItemAdding(properties);
        }

        public override void ItemAdded(SPItemEventProperties properties)
        {
            base.ItemAdded(properties);
        }

        public override void ItemUpdating(SPItemEventProperties properties)
        {
            base.ItemUpdating(properties);
        }

        public override void ItemUpdated(SPItemEventProperties properties)
        {
            base.ItemUpdated(properties);
        }

    }
}

File: VB/Code/SPTemplate.EventHandlers/SampleEventHandler.vb
Imports System.Collections.Generic
Imports System.Text
Imports Microsoft.SharePoint
Imports System.Web 
Imports $customUsing$.Common
Imports $customUsing$.Exceptions


Public Class SampleEventHandler
    Inherits SPItemEventReceiver

    Private current As HttpContext
    Public Sub New()
        current = HttpContext.Current
    End Sub

    Public Overrides Sub ItemAdding(ByVal properties As SPItemEventProperties)
        MyBase.ItemAdding(properties)
    End Sub

    Public Overrides Sub ItemAdded(ByVal properties As SPItemEventProperties)
        MyBase.ItemAdded(properties)
    End Sub

    Public Overrides Sub ItemUpdating(ByVal properties As SPItemEventProperties)
        MyBase.ItemUpdating(properties)
    End Sub

    Public Overrides Sub ItemUpdated(ByVal properties As SPItemEventProperties)
        MyBase.ItemUpdated(properties)
    End Sub

End Class
